cmd/allocator/main.go

```diff
@@ -143,7 +140,7 @@ func (h *httpHandler) allocateHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	allocation := h.agonesClient.AllocationV1alpha1().GameServerAllocations(h.namespace)
+	allocation := h.agonesClient.AllocationV1alpha1().GameServerAllocations(gsa.Namespace)
```

Comment:
Super minor nit - we've standardised on `{foo}.ObjectMeta.{var}` rather than going the shortcut version. 

```suggestion
	allocation := h.agonesClient.AllocationV1alpha1().GameServerAllocations(gsa.ObjectMeta.Namespace)
```

No major reason, just picked a version arbitrarily at the beginning, would be nice to keep it through :+1: 

Everything else looks great - weird that it's failing.

Reply:
I figured. I am getting 403 because I am trying to create gameserverallocation from a different namespace. It should be agones-system namespace. Actually this PR is not needed. The allocator service is correctly forwarding the allocation request to gameserverallocation api server extension.

Reply:
To simplify I will pass the agones-system as a namespace constant.